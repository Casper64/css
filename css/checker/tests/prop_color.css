.multiple_values {
    color: green, red;
}

.unsupported_function {
    color: bla(255, 255, 255);
}