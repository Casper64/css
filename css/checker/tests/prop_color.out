css/checker/tests/prop_color.css:2:12: error: property "color" only expects 1 value!
    1 | .multiple_values {
    2 |     color: green, red;
      |            ~~~~~~~~~~~
    3 | }
    4 |
css/checker/tests/prop_color.css:6:12: error: unsupported function "bla" for property "color".
Did you mean `rgb`, `rgba`, `hsl`?
    4 | 
    5 | .unsupported_function {
    6 |     color: bla(255, 255, 255);
      |            ~~~~~~~~~~~~~~~~~~
    7 | }